Bottom: 2c8828a3e3e3b22fcc857f7cb2c2fb36342d8ba6
Top:    172610d312af1b6d09887af0b4d081294db3c4ec
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-11 10:33:16 -0700

Refresh of iohandler-fallback

---
diff --git a/lldb/source/Core/IOHandler.cpp b/lldb/source/Core/IOHandler.cpp
index a69f93614e7..46be29f6fbf 100644
--- a/lldb/source/Core/IOHandler.cpp
+++ b/lldb/source/Core/IOHandler.cpp
@@ -70,6 +70,10 @@
 
 using namespace lldb;
 using namespace lldb_private;
+using llvm::None;
+using llvm::Optional;
+using llvm::StringRef;
+
 
 IOHandler::IOHandler(Debugger &debugger, IOHandler::Type type)
     : IOHandler(debugger, type,
@@ -306,10 +310,6 @@ void IOHandlerEditline::Deactivate() {
   m_delegate.IOHandlerDeactivated(*this);
 }
 
-using llvm::None;
-using llvm::Optional;
-using llvm::StringRef;
-
 // Split out a line from the buffer, if there is a full one to get.
 static Optional<std::string> SplitLine(std::string &line_buffer) {
   size_t pos = line_buffer.find('\n');
